dashboard/src/features/orgs/projects/overview/components/OverviewMetrics/OverviewMetrics.tsx

```diff
@@ -1,54 +1,132 @@
-  const { data, loading, error } = useGetProjectMetricsQuery({
+  const remoteProjectGQLClient = useRemoteApplicationGQLClient();
+
+  const {
+    data: {
+      allUsers: { aggregate: { count: allUsers = 0 } = {} } = {},
+      dailyActiveUsers: {
+        aggregate: { count: dailyActiveUsers = 0 } = {},
+      } = {},
+      monthlyActiveUsers: {
+        aggregate: { count: monthlyActiveUsers = 0 } = {},
+      } = {},
+      filesAggregate: {
+        aggregate: { sum: { size: totalStorage = 0 } = {} } = {},
+      } = {},
+    } = {},
+  } = useGetUserProjectMetricsQuery({
+    client: remoteProjectGQLClient,
+    variables: {
+      startOfMonth: startOfMonth(new Date()),
+      today: startOfDay(new Date()),
+    },
+    skip: !project,
+  });
+
+  const {
+    data: {
+      totalRequests: { value: totalRequestsInLastFiveMinutes = 0 } = {},
+    } = {},
+  } = useGetProjectRequestsMetricQuery({
+    variables: {
+      appId: project.id,
+      from: formatISO(subMinutes(new Date(), 6)), // 6 mns earlier
+      to: formatISO(subMinutes(new Date(), 1)), // 1 mn earlier
+    },
+    skip: !project,
+    pollInterval: 1000 * 60 * 5, // Poll every 5 minutes
+  });
+
+  const {
+    data: {
+      functionsDuration: { value: functionsDuration = 0 } = {},
+      totalRequests: { value: totalRequests = 0 } = {},
+      postgresVolumeUsage: { value: postgresVolumeUsage = 0 } = {},
+      egressVolume: { value: egressVolume = 0 } = {},
+    } = {},
+    loading,
+    error,
+  } = useGetProjectMetricsQuery({
     variables: {
-      appId: project?.id,
+      appId: project.id,
       subdomain: project?.subdomain,
       from: new Date(now.getFullYear(), now.getMonth(), 1),
     },
-    skip: !project?.id,
+    skip: !project,
   });
 
   const cardElements: MetricsCardProps[] = [
     {
-      label: 'CPU Usage Seconds',
-      tooltip: 'Total time the service has used the CPUs',
-      value: prettifyNumber(data?.cpuSecondsUsage?.value || 0),
+      label: 'Monthly Active Users',
+      tooltip: 'Unique users active this month',
+      value: prettifyNumber(monthlyActiveUsers),
+    },
+    {
+      label: 'Daily Active Users',
+      tooltip: 'Unique users active today',
+      value: prettifyNumber(dailyActiveUsers),
+    },
+    {
+      label: 'All Users',
+      tooltip: 'Total registered users',
+      value: prettifyNumber(allUsers),
+    },
+    {
+      label: 'Storage',
+      tooltip: 'Total size of stored files in the storage service',
+      value: prettifySize(totalStorage || 0),
     },
     {
       label: 'Total Requests',
-      tooltip:
-        'Total amount of requests your services have received excluding functions',
-      value: prettifyNumber(data?.totalRequests?.value || 0, {
-        numberOfDecimals: data?.totalRequests?.value > 1000 ? 2 : 0,
+      tooltip: 'Total service requests (excluding functions)',
+      value: prettifyNumber(totalRequests || 0, {
+        numberOfDecimals: totalRequests > 1000 ? 2 : 0,
```

Comment:
I am a bit dumb and I am failing to understand how you are getting this number. I only see the query that limits this to 5 minutes

Reply:
I'm doing two queries:

1. This query gets total requests in the last 5 minutes which I'm polling once every 5mns

https://github.com/nhost/nhost/pull/3052/files#diff-de881837e53f594075bb725282b02e92c2cb281f8f6a438fdbaa2e3254907fd1R47

```ts
const {
  data: {
    totalRequests: { value: totalRequestsInLastFiveMinutes = 0 } = {},
  } = {},
} = useGetProjectRequestsMetricQuery({
  variables: {
    appId: project.id,
    from: formatISO(subMinutes(new Date(), 6)), // 6 mns earlier
    to: formatISO(subMinutes(new Date(), 1)), // 1 mn earlier
  },
  skip: !project,
  pollInterval: 1000 * 60 * 5, // Poll every 5 minutes
});
```

2. This other one pulls totalRequests and other metrics only once for the current month

https://github.com/nhost/nhost/pull/3052/files#diff-de881837e53f594075bb725282b02e92c2cb281f8f6a438fdbaa2e3254907fd1R62

```ts
const {
  data: {
    functionsDuration: { value: functionsDuration = 0 } = {},
    totalRequests: { value: totalRequests = 0 } = {},
    postgresVolumeUsage: { value: postgresVolumeUsage = 0 } = {},
    egressVolume: { value: egressVolume = 0 } = {},
  } = {},
  loading,
  error,
} = useGetProjectMetricsQuery({
  variables: {
    appId: project.id,
    subdomain: project?.subdomain,
    from: new Date(now.getFullYear(), now.getMonth(), 1),
  },
  skip: !project,
});
```